src/main/java/org/sunbird/operationalreports/service/OperationalReportServiceImpl.java

```diff
@@ -367,6 +372,30 @@ private long getParsedDate(String reportExpiryDate) throws ParseException {
         return reportExpiryDateMillis;
     }
 
+
+    public SBApiResponse getFileInfo(String authToken) {
+        logger.info("Inside the getFileInfo()");
+        String userId = accessTokenValidator.fetchUserIdFromAccessToken(authToken);
+        if (null == userId) {
+            throw new HttpServerErrorException(HttpStatus.INTERNAL_SERVER_ERROR, "UserId Couldn't fetch from auth token");
```

Comment:
this API must return response object with proper error details instead of throwing exception..

Reply:
have created a response object and returned.